planet/clients/orders.py

```diff
@@ -271,18 +274,67 @@ async def download_asset(self,
                              progress_bar=progress_bar)
         return dl_path
 
+    @staticmethod
+    def validate_checksum(manifest_data: dict, filenames: list, checksum: str):
```

Comment:
@mkshah605 after thinking about this a little more, I believe we will be doing ourselves and our users a favor if we remove this static method from the OrdersClient class and turn it into a stand-alone function in the orders module.

The implementation doesn't require a client object, it only requires a manifest document and a directory of downloaded assets. The function might be useful outside the context of downloading an order, for example: a user might want to re-validate their data after copying their downloaded assets to another computer.